tests/new_api/test_device.py

```diff
@@ -29,6 +29,7 @@
     _supports_adjoint,
     accepted_observables,
     adjoint_measurements,
+    adjoint_observables,
```

Comment:
Should we test `_assert_adjdiff_no_projectors` instead?

Reply:
See above comment.